scripts/pr-tests-compose.sh

```diff
@@ -74,10 +74,22 @@ elif [ "$1" = "down" ]; then
 
 else
 
-    # Shutdown and clean the docker instance
+    # Shutdown and clean the docker instance (remove "osrd" images too)
+    osrd_images=$(docker compose \
+        -p "osrd-pr-tests" \
+        -f "docker/docker-compose.pr-tests.yml" \
+        images \
+        core editoast osrdyne gateway front \
+        | grep "pr-tests" \
+        | sed -e "s/[[:space:]]\+/ /g" \
+        | cut -f4 -d ' ')
```

Comment:
Just to remove some of the magic of it...

```suggestion
        | cut --fields=4 --delimiter=' ')
```

and actually, most of the above code too `docker compose --file ... --profile ...`, `sed --expression` (you might also use `--regexp-extended` to avoid to escape `\+`)...

Reply:
`awk '/pr-tests/ {print $4}'`

Not really magic, no grep, sed then cut required :p

Although I'd prefer having docker compose outputting some json and using jq

Reply:
Well, it was a general comment on my general aversion 👼 for short options in scripts.

Reply:
And it was a general comment about my love for awk :p

Reply:
Agreed for the long options.
Agreed for moving to `awk`.

Before changing this:
Agree that `jq` is even better, but I didn't want to ask for that dependency.
If you all think it's better, it sure won't bother me.

Reply:
I don't think jq will pose a problem for most people. If it does, we can always circle back to awk

Reply:
Move to `jq` done in the amended commit.

Didn't expend all the existing options for docker compose, which should still be OK.